srtp/srtp.c

```diff
@@ -3202,6 +3206,9 @@ static void srtp_calc_aead_iv_srtcp(srtp_session_keys_t *session_keys, v128_t *i
     in.v16[0] = 0;
     memcpy(&in.v16[1], &hdr->ssrc, 4); /* still in network order! */
     in.v16[3] = 0;
+    if (seq_num & 0x80000000) { /* the most significant bit should be zero */
```

Comment:
Given this is a constant we're not using elsewhere, I'm not terribly favorable to #defining it.  Either way, it's a line of code that somebody could break.  But, looking at it again, I do wonder if we should tack on an 'L' on the end of that.  And, perhaps we should do the same in datatypes.c?